analysis/geospatial_check.sql

```diff
@@ -0,0 +1,27 @@
+DROP IF EXISTS geospatial_check_table;
+CREATE TABLE IF NOT EXISTS geospatial_check_table(
+    v character varying,
+    result character varying
+);
+
+DELETE FROM geospatial_check_table
+WHERE v = :'ccp_v';
```

Comment:
very curious the use of the colon here. Which is outside my knowledge about sql. Not sure if it has anything to do with the script fail to run on my local.

Reply:
ok let me check this error

Reply:
sorry that's my bad, forgot to pass the environment variables when calling psql, modified it